flower-shop/src/main/java/com/flower/shop/application/authentication/AuthenticationService.java

```diff
@@ -33,6 +37,9 @@ public AuthenticationResponse register(RegisterRequest request) {
         var jwtToken = jwtService.generateToken(user);
         return AuthenticationResponse.builder()
                 .token(jwtToken)
+                .name(user.getFirstName())
```

Comment:
czy w registerze te dane są potrzebne? jak nie to można zwrócić nulla, czyli w ogóle nie ustawiać tych pól i w authenticationResponse użyć adnotacji do ignorowania nulli w jsonie

Reply:
ale po co ci to przy rejestracji